### ISSUE TITLE
Unable to have travis ci run test:browser

### ISSUE THREAD

**Issue by winksaville** [winksaville/nop-ts#1]
For having a test-on-travis only run test:node and not both test:node and test:browser like test does.

I'm getting tried a couple different configurations, see travis runs [20](https://travis-ci.org/winksaville/nop-ts/builds/208383049) and [21](https://travis-ci.org/winksaville/nop-ts/builds/208391473), both generate the same error:
```
The ChromeDriver could not be found on the current PATH. Please download the latest version of the ChromeDriver from http://chromedriver.storage.googleapis.com/index.html and ensure it can be found on your PATH.
```

Run 20 was configured from [here](http://blog.500tech.com/setting-up-travis-ci-to-run-tests-on-latest-google-chrome-version/) and Run 21 was configured from [here](http://stackoverflow.com/questions/20364674/chromedriver-on-travis-ci)

**Comment by winksaville** [comment]
The chromedriver needed to be installed, which was the reason for the ChromeDriver not being found, the following was added to `before_script` section:

  - LATEST=$(wget -q -O - https://chromedriver.storage.googleapis.com/LATEST_RELEASE) ; wget http://chromedriver.storage.googleapis.com/$LATEST/chromedriver_linux64.zip
  - unzip chromedriver_linux64.zip && sudo ln -s $PWD/chromedriver /usr/local/bin/chromedriver

But Travis is now hanging at the end of the tests, it appears Alsatian isn't stopping see [this build](https://travis-ci.org/winksaville/nop-ts/builds/209453090):
```
$ yarn test:node & yarn test:browser 
yarn test:browser v0.17.8
yarn test:node v0.17.8
$ alsatian build/test.js --tap 
$ alsatian build/test-web.js --tap 
TAP version 13
1..3
# FIXTURE NopTests
ok 1 test nop type is function
ok 2 test nop return value is not defined
ok 3 test a js file can import and call nop
setupFixture:+
setupFixture: 1
setupFixture: 2
TAP version 13
1..2
setupFixture:-
# FIXTURE Browser nop tests
testWdNop:+
testWdNop: 1
testWdNop:-
ok 1 wd: nop which is defined in a TS file
testWdCallNop:+
testWdCallNop: 1
testWdCallNop:-
ok 2 wd: callNop which is defined in a JS file
teardownFixture:+
teardownFixture: 1
teardownFixture:-
No output has been received in the last 10m0s, this potentially indicates a stalled build or something wrong with the build itself.
Check the details on how to adjust your build configuration on: https://docs.travis-ci.com/user/common-build-problems/#Build-times-out-because-no-output-was-received
The build has been terminated
```

**Comment by winksaville** [comment]
Fixed with 77844e6103fe, spawn needed to be used instead of exec.
